boards/DCM/Inc/App/App_DcmWorld.h

```diff
@@ -28,7 +30,10 @@ struct DcmWorld *App_DcmWorld_Create(
     struct BrakeLight *       brake_light,
     struct Buzzer *           buzzer,
     struct ErrorTable *       error_table,
-    struct Clock *            clock);
+    struct Clock *            clock,
+
+    bool (*is_buzzer_on)(struct DcmWorld *),
+    void (*buzzer_callback)(struct DcmWorld *));
```

Comment:
LGTM did you just want to change the order here as a NIT?

Reply:
Do we currently have a convention for this?

Reply:
No i don't think so it just looks nicer to group the buzzer stuff together than have part of it after the clock structure

Reply:
I prefer this cuz it splits the members in world from the callback related signals